modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_dependency 'claims_api/application_controller'
+require 'jsonapi/parser'
+
+module ClaimsApi
+  module V0
+    module Forms
+      class DisabilityCompensationController < ApplicationController
+        skip_before_action(:authenticate)
+        # before_action :validate_json_api_payload
+
+        def form_526
+          auto_claim = ClaimsApi::AutoEstablishedClaim.create(
+            status: ClaimsApi::AutoEstablishedClaim::PENDING,
+            auth_headers: auth_headers,
+            form_data: form_attributes
+          )
+          auto_claim.form.to_internal
+
+          ClaimsApi::ClaimEstablisher.perform_async(auto_claim.id)
+
+          render json: auto_claim, serializer: ClaimsApi::AutoEstablishedClaimSerializer
+        rescue RuntimeError => e
+          render json: { errors: e.message }, status: 422
+        end
+
+        private
+
+        def validate_json_api_payload
+          JSONAPI.parse_resource!(params)
+        end
+
+        def form_attributes
+          params[:data][:attributes]
+        end
+
+        def target_veteran
+          vet = ClaimsApi::Veteran.from_headers(request.headers, true)
+          vet.loa = { current: 2 }
+          vet
+        end
```

Comment:
This seems like a code smell to me - injecting a level of assurance. The purpose of propagating a level of assurance is to basically assert an attribute about the user's authentication. In this case the user (i.e. the veteran) is not authenticating, right? But we are still asserting a LOA as if they did. We should not make up authentication data just because it's a required field to make the API call work. 

Have we established with BGS or whoever the downstream system is whether they have an integration pattern for auth headers to represent third-party submissions, i.e. not originating from a veteran acting as themself logged in to VA.gov or eBenefits?

Reply:
Currently the implementation is with EVSS which requires the loa2 and loa3 access, we'll take another look at this tomorrow and see if there's a way around needing to use it and or how the front end team is doing it. Thanks

Reply:
I went ahead and removed the loa references from this PR, we have the responses mocked so we can merge and deploy this to have people test this in dev-api.va.gov while we solve the loa issue for testing in staging and production later on.